astro/src/content/blog/announcing-fusionauth-1-49.mdx

```diff
@@ -0,0 +1,93 @@
+
+## Elasticsearch Reindexing Progress
+
+I don't know about you, but I like progress indicators. I remember, years ago, watching progress indicators all the time, from network downloads to disk defragmentation programs. (RIP `Defrag`.) We all have faster internet and disk now, so hopefully those indicators are a thing of the distant past for you.
+
+But if you use FusionAuth and [reindex your Elasticsearch (or Opensearch) index](/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch), before this release FusionAuth didn't do a great job of reporting progress. It only reported back every 250k objects, which depending on the data in the index, could give the impression that progress has halted.
+
+In this release and going forward, FusionAuth will report reindexing progress more regularly in order to spare everyone some frustration.
+
+## Email Link Workflow Improvements
+
+Speaking of frustration, FusionAuth has dealt with a lot of issues around [our magic links functionality](/docs/lifecycle/authenticate-users/passwordless/magic-links). This seems pretty straightforward:
+
+* create a customizable, localizable email template: check!
+* create a one-time use, random, high entropy string tied to a user account: check!
+* send the email to an account: check!
+
+If the user clicks on the link in the email, FusionAuth is assured they own the email address and should be logged in.  
+
+But, it's not quite that simple.
+
+Link checkers, which are common in corporate email settings, defeat this in the name of preventing phishing. They retrieve all the links in the email to make sure they are safe, invalidating the one-time code.
+
+FusionAuth tried to solve the email link checker issue by taking the GET request from the user clicking on a link in an email client, and returning an HTML form including the one-time code. After the form loads, FusionAuth submits it via JavaScript, which converts the `GET` into a `POST`. 
+
+In most cases, FusionAuth found web crawlers or link checkers will perform a `HEAD` request, perhaps followed by a `GET` request. If that is all the link checker does, the above strategy avoids one-time code invalidation. The user will get the desired result: they'll be logged in.
+
+However, FusionAuth clients have demonstrated that Outlook Safelinks both performs the `GET` and submits the form. This means that the `POST` request happens, which causes the one-time code to to be used before the user is presented with the page. The link is then invalid and when the user clicks it, they arrive at a screen indicating that.
+
+As far as the team could discover, Microsoft does not document any legitimate way to stop this or indicate that the link should not be followed and submitted. This makes sense because documented 'safe link' attributes or some other method of skipping the link check would allow a malicious actor to defeat them.
```

Comment:
```suggestion
As far as the team could discover, Microsoft does not document any legitimate way to stop this or indicate that the link should not be followed and submitted. This makes sense because documented `safe link` attributes or some other method of skipping the link check would allow a malicious actor to defeat them.
```

Reply:
Did you want this highlighted?